### PR TITLE
Add contributing article and style guide

### PR DESCRIPTION
**Pull request by harterrt**
Rendered documentation can be previewed here:
https://harterrt.github.io/firefox-data-docs/meta/contributing.html

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/firefox-data-docs/41)

### CODE REVIEW

File: meta/contributing.md [hunk 1]
```diff
@@ -1,6 +1,99 @@
...
+
+## Style Guide
+
+Articles should be written in
+[Markdown](https://daringfireball.net/projects/markdown/syntax)
+(not [AsciiDoc](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/)).
+Markdown is usually powerful enough and is a more common technology than Asciidoc.
+
+Limit lines to **100 characters** where possible.
+Try to split lines at the end of sentances.
```
Comment (mreid-moz) [inline meta/contributing.md]: *sentences

File: meta/contributing.md [hunk 2]
```diff
@@ -1,6 +1,99 @@
...
+This makes it easier to reorganize your thoughts later.
+
+This documentation is meant to be read digitally.
+Keep in mind that people read digital content much differently than other media.
+Specifically, readers are going to skim your writing,
+so make it easy to identify important information
```
Comment (mreid-moz) [inline meta/contributing.md]: missing period at the end of this sentence

File: meta/contributing.md [hunk 3]
```diff
@@ -1,6 +1,99 @@
...
+
+The documentation is rendered with [Gitbook](https://www.gitbook.com/)
+To build the documentation locally,
+you'll need to install the gitbook-cli app with npm:
+
+```bash
+npm install gitbook-cli -g
+```
+
+and install the gitbook plugins with
```
Comment (mreid-moz) [inline meta/contributing.md]: missing `:` at the end of the line

File: meta/contributing.md [hunk 4]
```diff
@@ -1,6 +1,99 @@
...
+so make it easy to identify important information
+
+Use **visual markup** like **bold text**, `code blocks`, and section headers.
+Avoid long paragraphs.
+Short paragraphs that describe one concept each makes finding important information easier.
+
+
+## Review
+
+Once your happy with your contribution, please open a PR and flag @harterrt for review.
```
Comment (mreid-moz) [inline meta/contributing.md]: s/your/you're/

File: meta/contributing.md [hunk 5]
```diff
@@ -1,6 +1,99 @@
...
+to publish new changes.
+
+This script depends on 
+[ghp-import](https://github.com/davisp/ghp-import).
+
+Keep in mind that this will deploy the docs to your `origin` repo.
+If you're working from a fork (which you should be),
+`deploy.sh` will update the docs hosted from your fork - not the production docs.
+
+# Colophone
```
Comment (mreid-moz) [inline meta/contributing.md]: I think "colophon", though this word is new to me :)

File: meta/contributing.md [hunk 6]
```diff
@@ -1,6 +1,99 @@
...
+[docs.telemetry.mozilla.org](https://docs.telemetry.mozilla.org).
+However, we store the documentation in raw text files in the
+[firefox-data-docs repo](https://github.com/mozilla/firefox-data-docs).
+To begin contributing to the docs, fork the firefox-data-docs repo.
+
+## Building the Documentation
+
+The documentation is rendered with [Gitbook](https://www.gitbook.com/)
+To build the documentation locally,
+you'll need to install the gitbook-cli app with npm:
```
Comment (mreid-moz) [inline meta/contributing.md]: consider linking `npm` to the npm installation instructions: https://www.npmjs.com/get-npm

Comment (harterrt) [reply meta/contributing.md]: done

File: meta/contributing.md (hunk 3 above)
Comment (harterrt) [reply meta/contributing.md]: done

File: meta/contributing.md (hunk 1 above)
Comment (harterrt) [reply meta/contributing.md]: https://github.com/mozilla/firefox-data-docs/issues/42

**Pull request merged**